### DISCUSSION TITLE
Can't get verified since i'm too far from campus

### DISCUSSION THREAD

**Discussion by NoelJasik** [community/community#147804] [GitHub Education]
### Hi everyone,

Hey, so I've tried to reapply like 3 times to get into the program, every time i get rejected because I'm too far away from campus, but i'm only 50kms away, I drive there, and I also got a holiday break. I could try sending the proof at school when I be there in january, but I can't even take a picture easliy with my phone camera as it seems only the front one does work

Any idea on how to get verfied?

**Comment by ghost** [comment]
I am facing the same issue. 
thats why Ima writin to seek clarification regardin the repeated rejection of my application for the GitHub Student Developer Pack.

I also have submitted my documents 10 times, and despite my efforts, my application keeps gettin rejected. The primary reason for the rejections appears to be that I am not currently on campus. I would like to inform uh that I am on winter vacation and will not be returning to campus until January 27th, 2025.

To support my case, I have provided an official notification from my college, confirming my current status. However, I am still facin rejections.

Could y kindly provide guidance on how I can proceed with my application, or let me know if there is any additional information or documentation I need to submit in order to get approved?

**Comment by ProfTAB** [comment]
I have an issue that I'm a PhD student, and I don't go to campus or have coursework. I work in a different country from my school, and only go there once a year to renew my library card.

**Comment by dantdmdarealguy** [comment]
I had this same issue. I just put "other" as the reason, explained about semester vacation, and then attached a picture of my campus calendar with proof of vacation days on it. That worked and I got accepted.

**Reply by ProfTAB** [reply]
I'll have to give that a try. Otherwise it'll have to wait until the next in-person visit.
